src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,26 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type SingleTodo = {
+  todo: Todo
+};
+
+export const TodoInfo: React.FC<SingleTodo> = ({ todo }) => (
+  <>
+    <article className={classNames(
+      'TodoInfo',
+      {
+        'TodoInfo--completed': todo.completed,
+      },
+    )}
+    >
+      <h2 className="TodoInfo__title">{`${todo.title}`}</h2>
```

Comment:
the title is already a string
also, you should move the text to the new line for improving readability
```suggestion
      <h2 className="TodoInfo__title">
        {todo.title}
      </h2>
```